admin/new-relic-admin.php

```diff
@@ -94,37 +106,46 @@
 					<input class="button-primary" type="submit" value="Submit" name="rtp-relic-get-browser-submit">
 				</p>
 			</form>
-			<form id="rtp-relic-add-account" class="<?php echo $new_relic_form; ?>" action="options.php" method="POST" enctype="multipart/form-data">
+			<form id="rtp-relic-add-account" class="<?php echo esc_attr( $new_relic_form ); ?>" action="options.php" method="POST" enctype="multipart/form-data">
 				<?php
+				wp_nonce_field( 'relic_options_nonce_action', 'submit_new_relic' );
 				settings_fields( 'relic_options_settings' );
+				$site_url = site_url();
+
+				$disallowed = array( 'http://', 'https://' );
+				foreach ( $disallowed as $d ) {
+					if ( strpos( $site_url, $d ) === 0 ) {
+						$site_url = str_replace( $d, '', $site_url );
+					}
+				}
 				?>
 				<table class="form-table">
 					<tbody>
 						<tr>
 							<th scope="row"><label for="relic-account-name">Account Name<span class="description"> (required)</span></label></th>
 							<td>
-								<input type="text" name="relic-account-name" id="relic-account-name" class="regular-text" value="<?php echo $_SERVER['SERVER_NAME']; ?>">
+								<input type="text" name="relic-account-name" id="relic-account-name" class="regular-text" value="<?php echo esc_attr( $site_url ); ?>">
 								<span id="relic-account-name_error" class="form_error"></span>
 							</td>
 						</tr>
 						<tr>
 							<th scope="row"><label for="relic-account-email">Email<span class="description"> (required)</span></label></th>
 							<td>
-								<input type="text" name="relic-account-email" id="relic-account-email" class="regular-text" value="<?php echo $current_user->user_email ?>">
+								<input type="text" name="relic-account-email" id="relic-account-email" class="regular-text" value="<?php echo esc_html( $current_user->user_email ); ?>">
 								<span id="relic-account-email_error" class="form_error"></span>
 							</td>
 						</tr>
 						<tr>
 							<th scope="row"><label for="relic-first-name">First Name<span class="description"> (required)</span></label></th>
 							<td>
-								<input type="text" name="relic-first-name" id="relic-first-name" class="regular-text" value="<?php echo $current_user->user_firstname ?>">
+								<input type="text" name="relic-first-name" id="relic-first-name" class="regular-text" value="<?php echo esc_html( $current_user->user_firstname ); ?>">
```

Comment:
For attributes, use `esc_attr()` to escape

Reply:
i solved this issue. please review changes.